wiki/Help_Centre/Website/id.md

```diff
@@ -0,0 +1,118 @@
+---
+tags:
+  - site
+  - block
+  - forum
+  - ticket
+  - appearance
+  - message
+  - page
+  - profile
+  - missing
+  - stats
+  - situs web
+  - pemblokiran
+  - tiket layanan dukungan
+  - tampilan
+  - pesan
+  - laman profil
+  - statistik
+---
+
+# Situs web
+
+*Laman utama: [Pusat bantuan](/wiki/Help_Centre)*
+
+Laman ini akan membahas berbagai permasalahan dan pertanyaan yang sering diutarakan seputar situs web osu! secara garis besar.
+
+## Masalah-masalah umum
+
+### Saya sudah menunggu hingga lebih dari 2 minggu, namun sampai saat ini saya belum mendapatkan kabar apa-apa seputar tiket layanan dukungan (*support ticket*) milik saya!
+
+**Pada umumnya, hal ini disebabkan oleh banyaknya tiket layanan dukungan (*support ticket*) yang sedang kami proses.**
+
+Apabila kamu ingin mengetahui status tiket layanan dukungan (support ticket) milikmu, kamu dipersilahkan untuk kembali menghubungi tim layanan dukungan kami asalkan masih dalam batas yang sewajarnya. Kamu juga dapat menanyakan hal yang sama melalui akun Twitter [@osusupport](https://twitter.com/osusupport) milik kami.
+
+Dalam **kasus-kasus tertentu** yang utamanya melibatkan pelanggaran-pelanggaran berat, tim layanan dukungan kami dapat memberitahukan kamu bahwa tiket layanan dukungan (support ticket) milikmu tidak akan ditindaklanjuti lebih lanjut. Apabila kamu tidak merasa pernah melakukan pelanggaran apapun, kamu tidak perlu khawatir dan hanya butuh untuk lebih bersabar!
+
+### Apakah saya dapat memblokir pengguna lain?
+
+**Tentu saja bisa.**
+
+Apabila kamu merasa tersinggung atau terlecehkan dengan pesan-pesan yang dikirim oleh pengguna-pengguna tertentu baik itu melalui pesan pribadi (*private message*), forum, ataupun kanal obrolan dalam permainan (*in-game chat*), kamu dapat memblokir mereka dengan mengikuti langkah-langkah berikut.
+
+#### Memblokir pengguna lain untuk dapat menghubungimu melalui *in-game chat*
```

Comment:
```suggestion
#### Memblokir pengguna lain dari menghubungimu melalui *in-game chat*
```

Reply:
did sth else here (and the subsequent points)